packages/account/src/providers/provider.test.ts

```diff
@@ -867,11 +867,14 @@ describe('Provider', () => {
     request.addCoinOutput(receiver.address, 500, provider.getBaseAssetId());
     request.addResources(resources);
 
+    const moreResources = await sender.getResourcesToSpend(quantities);
```

Comment:
A `getCoinsToSpend` will now truncate the response to `maxInputs`. So our validation is redundant when request inputs have been added by a single `getCoinsToSpend` query. So we should manually add more inputs for our validation to execute, or call the query again.

Reply:
I would add a comment about this.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3332/commits/6bbcae0b53a05874c53e0f57e5b1c103f01d0406